.github/workflows/ci.yml

```diff
@@ -0,0 +1,185 @@
+        with:
+          python-version: "${{ matrix.python-version }}"
+      - name: "Run unittest"
+        run: "poetry run invoke unittest --failfast --keepdb"
+      - name: "Run coverage"
+        run: "poetry run invoke unittest-coverage"
+    needs:
+      - "check-migrations"
+  tests-mysql:
+    runs-on: "ubuntu-20.04"
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [ "3.6", "3.7", "3.8", "3.9" ]
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+      NAUTOBOT_DB_USER: "root"
+      NAUTOBOT_DB_PASSWORD: "decinablesprewad"
+      NAUTOBOT_DB_HOST: "127.0.0.1"
+      NAUTOBOT_DB_PORT: "3306"
+      NAUTOBOT_DB_ENGINE: "django.db.backends.mysql"
+    services:
+      mysql:
+        image: "mysql:8"
+        env:
+          MYSQL_ROOT_PASSWORD: "decinablesprewad"
+          MYSQL_PASSWORD: "decinablesprewad"
+          MYSQL_DATABASE: "nautobot"
+        ports:
+          - "3306:3306"
+      redis:
+        image: "redis:6-alpine"
+        ports:
+          - "6379:6379"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v2"
+        with:
+          python-version: "${{ matrix.python-version }}"
+      - name: "Install Required Dependencies for MySql Client"
+        run: "sudo apt-get update && sudo apt-get install --no-install-recommends -y default-libmysqlclient-dev gcc"
+      - name: "Install MySql Dependencies"
+        run: "poetry install --no-interaction --no-root --extras mysql"
+      - name: "Run unittest"
+        run: "poetry run invoke unittest --failfast --keepdb"
+      - name: "Run coverage"
+        run: "poetry run invoke unittest-coverage"
+    needs:
+      - "check-migrations"
+  integration-test:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+      NAUTOBOT_DB_USER: "nautobot"
+      NAUTOBOT_DB_PASSWORD: "decinablesprewad"
+      NAUTOBOT_DB_ENGINE: "django.db.backends.postgresql"
+      NAUTOBOT_SELENIUM_URL: "http://localhost:4444/wd/hub"
+    services:
+      postgres:
+        image: "postgres:13-alpine"
+        env:
+          POSTGRES_USER: "nautobot"
+          POSTGRES_PASSWORD: "decinablesprewad"
+          POSTGRES_DB: "nautobot"
+        ports:
+          - "5432:5432"
+      redis:
+        image: "redis:6-alpine"
+        ports:
+          - "6379:6379"
+      selenium:
+        image: "selenium/standalone-firefox:latest"
+        ports:
+          - "4444:4444"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v2"
+      - name: "Run Integration Tests"
+        # If NAUTOBOT_SELENIUM_HOST is set to 'localhost' or '127.0.0.1' the connection does not work
+        run: "NAUTOBOT_SELENIUM_HOST=`hostname -f` poetry run invoke integration-test --keepdb --append"
```

Comment:
Doesn't look like `--append` is serving any purpose here - in the travis.yml it was allowing for aggregation of the unit and integration test coverage numbers, but here (I guess because things are isolated in a way they aren't in Travis?) the integration-test coverage reported is much lower, clearly not aggregated/combined with the unit-test coverage.

Reply:
Hmm, I am not sure if I follow. But, I can remove it if not needed...

Reply:
Yeah append only really makes sense if the unit and integration tests share the same disk cache or run in the same stage. If that's not happening here, especially if containers are being used without a shared volume, then there is no need to have it.